build.gradle

```diff
@@ -157,12 +157,18 @@ repositories {
 // it is important to specify logback classic and core packages explicitly as libraries like spring boot
 // enforces it's own (older) version which is not recommended.
 def versions = [
-  reformLogging   : '3.0.2',
+  reformLogging   : '3.0.3',
   springBoot      : springBoot.class.package.implementationVersion,
   springfoxSwagger: '2.9.2',
   junit           : '5.3.1'
 ]
 
+configurations.all {
+  resolutionStrategy {
+    force "com.google.guava:guava:27.0-jre"
```

Comment:
Is it safe?

Reply:
Fair enough. It would be worth adding a comment explaining why we're forcing this version.

Reply:
in git annotations you probably will find the commit message explaining it but fair point - can also add a comment